native/packages/react-native-bpk-component-button/src/BpkButton.js

```diff
@@ -18,152 +18,44 @@
 
 import {
   View,
-  TouchableHighlight,
-  TouchableNativeFeedback,
   Platform,
 } from 'react-native';
 import React from 'react';
 import PropTypes from 'prop-types';
-import LinearGradient from 'react-native-linear-gradient';
-import difference from 'lodash/difference';
 
 import { withTheme } from 'react-native-bpk-theming';
 import BpkIcon from 'react-native-bpk-component-icon';
 import BpkText from 'react-native-bpk-component-text';
 
-import styles, { colorWhite } from './BpkButton-styles';
-
+import {
+  isTypeThemeable,
+  themeAttributesSupplied,
+  getStyleForElement,
+  iconPropType,
+  themePropType,
+  textStyle,
+} from './BpkButton-utils';
+
+import BpkButtonAndroidContainer from './layout/BpkButtonContainer.android';
```

Comment:
you should technically be able to avoid the `Platform.OS` check because the RN packager already conditionally imports file based on it's extension. I.e:

```
import BpkButtonContainer from './layout/BpkButtonContainer';
```
This will import `BpkButtonContainer.{platform}.js` based on current `Platform.OS`

Reply:
Didn't know about that, that's really cool. I guess it's the same as how storybook does it.

Reply:
Forgot a bout that, thanks!